csharp-basic-topics/AwaitInForEachLoops/AwaitInForEachLoops/AwaitInLoops.cs

```diff
@@ -0,0 +1,27 @@
+﻿
+
+namespace AwaitInForEachLoop
+{
+    public static class AwaitInLoops
+    {
+        public static async Task<List<int>> ResultAsync()
+        {
+            var numbers = new List<int> { 10, 20, 30, 40, 50, 60, 70, 80, 90 };
+
+            var result = new List<int>();
+
+            foreach (var number in numbers)
+            {
+                result.Add(number);
+
+                Console.WriteLine($"Processed {number}");
```

Comment:
Print the below line before Task.Delay(1000)
Console.WriteLine($"Processing {number}");

and move the line Console.WriteLine($"Processed {number}"); after Task.Delay(1000).
Also, we must add the number to the result list after it is processed.

Reply:
done